ruby-challenges.rb

```diff
@@ -3,32 +3,58 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
-reposts1 = 7
-# Expected output: '7 is odd'
-reposts2 = 42
-# Expected output: '42 is even'
-reposts3 = 221
-# Expected output: '221 is odd'
+    # Pseudocode:
+        # Create Ruby method named 'even_odd'
+        # Use the '.even?' method that checks to see if given number is even. If given number is even, method will return a string that says "is even."
+        # Use the '.odd?' method that checks to see if a given number is odd. If given number is odd, method will return a string that says "is odd."
+        # Output will be the given number using string interpolation with saying "#{number} is even" or "#{number is odd}"
+
+            def even_odd(number)
+                if number.even? 
+                    "#{number} is even"
+                else number.odd? 
+                    "#{number} is odd"
+                end
+            end
+
+            p even_odd(7) # output: "7 is odd"
+            p even_odd(42) # output: "42 is even"
+            p even_odd(221) # output: "221 is odd"
 
 
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. 
 # HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
-
-beatles_album1 = 'Rubber Soul'
-# Expected output: 'Rbbr Sl'
-beatles_album2 = 'Sgt Pepper'
-# Expected output: 'Sgt Pppr'
-beatles_album3 = 'Abbey Road'
-# Expected output: 'bby Rd'
+    # Pseudocode:
+            # Create Ruby method named 'remove_vowels' that takes in a string as the parameter
+            # Use '.delete' method to delete all uppercase and lowercase vowels within given string
+            # Output will be the given strings with all vowels removed
+    
+                def remove_vowels(string)
+                    string.delete("aeiou" || "AEIOU")
+                end
+                p remove_vowels('Rubber Soul')
+                p remove_vowels('Sgt Pepper')
+                p remove_vowels('Abbey Road')
 
```

Comment:
I see your idea using the logical operator, but it's not quite working how you want it to. If you test it on `beatles_album3`, it still doesn't remove that capital A. You have the right idea, but the logical operator is actually not necessary at all!

Reply:
I was so iffy about using that. Thank you for letting me know!